packages/runtime/runtime-definitions/src/compatUtils.ts

```diff
@@ -4,15 +4,15 @@
  */
 
 import { assert } from "@fluidframework/core-utils/internal";
-import { FlushMode } from "@fluidframework/runtime-definitions/internal";
```

Comment:
we don't usually put code in our definition packages, they are mainly mean to be just interfaces. there are utils packages where code lives. i also don't think most of this should move, there may be some common things are that reusable that could move, but in generally out compat enforcement will need to be layered just like everything else in fluid is layer, so the container runtime specific stuff should stay where it is